TopicZero/src/key_three/PrivateHospital.java

```diff
@@ -0,0 +1,23 @@
+package key_three;
+
+public class PrivateHospital implements Hospital {
+
+	@Override
+	public boolean acceptsMedicalInsurance() {
+		// TODO Auto-generated method stub

```

Comment:
Please try to remove all the TODO comments if you don't need to add more code below